### PR TITLE
Force decrypt_kernel to use python2

### LINKED ISSUE #11: Use Python3 instead of Python2
Use Python3 instead of Python2 throughout scripts/programs. This may include [Sandblaster](https://github.com/malus-security/sandblaster).

### LINKED ISSUE #12: Decrypt disk image step fails when python3 is implicitly used
When multiple versions of python are installed on the same system(i. e. python 2.7 and python 3), run_all script fails to decrypt disk image file because of a print statement. For now, a quick(and rather dirty) fix is modifying bin/decrypt_kernel to explicitly use python2(line 34).

### PR DESCRIPTION
**Pull request by andreicalin-georgescu**
Proposes a change to line 34 in bin/decrypt_kernel to explicitly use python2 when running get_lzss_section_offset.py script

### CODE REVIEW

**Comment by razvand** [comment]
@andreicalin-georgescu , it would be best if we used Python3 throughout the scripts. Could you please update the `get_lszz_section_offset.py` script to use Python3? And then call it using `python3` from `bin/decrypt_kernel`.

**Comment by andreicalin-georgescu** [comment]
@razvand I made the changes to the scripts. Everything seems to be working on my end.

File: bin/decrypt_kernel [hunk 1]
```diff
@@ -31,7 +31,7 @@ else
     warn_if_error
 fi
 
-lzss_offset=$(./get_lzss_section_offset.py "$kernelcache_decrypted")
+lzss_offset=$(python3 ./get_lzss_section_offset.py "$kernelcache_decrypted")
```
Comment (razvand) [inline bin/decrypt_kernel:34]: Please also update the shebang line in `bin/get_lzss_section_offset.py` to `#!/usr/bin/env python3`.

File: bin/get_lzss_section_offset.py [hunk 2]
```diff
@@ -19,11 +19,11 @@ def main():
             if len(buf) < 16:
                 break
             for i in range(0, 16):
-                if ord(buf[i]) == 0xff and i+4 < 16:
+                if buf[i] == 0xff and i+4 < 16:
                     chunk = struct.unpack("<I", buf[i+1:i+5])[0]
                     if chunk == 0xfeedface or chunk == 0xfeedfacf:
                         found = True
-                        print offset+i
+                        print (offset+i)
```
Comment (razvand) [inline bin/get_lzss_section_offset.py]: Use `print(offeset+i)`, remove blank before paranthesis.

**Review by razvand — CHANGES_REQUESTED** [review]
Add minor feedback on presentation and formatting.

**Comment by razvand** [comment]
@andreicalin-georgescu , please squash your commits into a single commit for this pull request.

**Comment by andreicalin-georgescu** [comment]
Presentation and formatting changed according to feedback. Hope everything's all right now. Sorry for the oversight on my part.

**Review by razvand — APPROVED** [review]
All good

**Pull request merged**